README.md

```diff
@@ -23,3 +23,50 @@ These differences include:
    As much as possible the code includes comments about these extensions.
 
  - Many data fields not used by CoreUpdate are omitted.
+ 
+## `serve-package`
+
+This project includes a very simple binary designed to serve a single Container Linux package on the local host. It is intended to be used as a manual updater, for when a server is for whatever reason not able to use a full-fledged CoreUpdate instance. Released binaries are available for each released version on the [releases page](https://github.com/coreos/go-omaha/releases). It can also be built from source using the provided Makefile:
+
+```bash
+make
+```
+
+The binary will be available in the `bin/` folder.
+
+It is recommended that the server be run directly on the machine you intend to update. Go to the [Container Linux Releases page](https://coreos.com/releases) and find the version number for the release you would like to update to. The update payload can be retrieved from
+
+```
+https://update.release.core-os.net/amd64-usr/<version>/update.gz
+```
+
+where `<version>` is the semantic version number you retrieved from the releases page. For example, `https://update.release.core-os.net/amd64-usr/1576.4.0/update.gz` would get the update payload required to update to Container Linux version 1576.4.0.
+
+Copy the update payload and the `serve-package` binary to the server you would like to update. The package server can be run as follows:
+
+```bash
+./serve-package --package-file update.gz --package-version <version>
+```
+
+By default, the server serves on `localhost:8000`. This can be modified using the `--listen-address` flag.
+
+Next, `update_engine` needs to be told to use the package server that was just setup. Add the following line to `/etc/coreos/update.conf`:
+
+```bash
+echo "SERVER=http://localhost:8000/v1/update" >> /etc/coreos/update.conf
+```
+
+Restart `update_engine` and tell it to look for an update immediately:
+
+```bash
+systemctl restart update-engine.service
+update_engine_client -check_for_update
+```
+
+If `locksmithd.service` is running, the machine will restart on it's own once it is ready to update to the latest version. Otherwise, watch the logs from `update-engine.service` to determine when the machine is done updating and is ready to restart:
```

Comment:
s/on it's own once it is ready to update/once it has updated/
s/the machine is done updating and is ready to restart/the update is complete and the machine is ready to restart/